pkg/trace/traceutil/trace.go

```diff
@@ -32,23 +31,6 @@ func GetEnv(root *pb.Span, t *pb.TraceChunk) string {
 	return ""
 }
 
-// GetAppVersion returns the first "version" tag found in trace t.
-// Search starts by root
-func GetAppVersion(root *pb.Span, t *pb.TraceChunk) string {
```

Comment:
I felt it would be best to have all of the logic around retrieving version data in one place. Although technically yes, this function did not have to be moved to resolve some cyclic imports.